index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Portofolio</title>
+</head>
+<body>
+  <div class="container">
+    <div class="toolbar">
+      <div class="navbar">
+        <span class="navbar_welcome">Welcome</span>
+        <div class="navbar_menu">
+          <div class="navbar_menu_button">
+            <img src="assets/menu_icon.svg" alt="menu_icon">
+          </div>
+        </div>
+        
+      </div>
+      
+    </div>
+    <div class="main_section">
+      <h1 class="main_section_h1">Hey there. I'm Koami.</h1>
+      <h2 class="main_section_h2">I'm a software developer</h2>
+      <p class="main_section_p">
+        professionally trained by microverse online university , 
+        I can help in custom software development (according to demand), 
+        website development, mobile application development,
+         network maintenance and many other IT areas.
+      </p>
+      <div class="main_section_social">
+        <a class="social_media_icon">
+          <img src="assets/github.svg" alt="github icon "/>
+        </a>
+        <a class="social_media_icon">
+          <img src="assets/angellist.svg" alt="angellist icon "/>
+        </a>
+        <a class="social_media_icon">
+          <img src="assets/Linkedin_icon.svg" alt="Linkedin icon "/>
+        </a>
+        <a class="social_media_icon" id="medium_icon">
+          <img src="assets/medium.svg" alt="medium icon "/>
+        </a>
+        <a class="social_media_icon" >
+          <img src="assets/twitter.svg" alt="twitter icon "/>
+        </a>
```

Comment:
- [ ] I think it will be better if you wrap your social icons in a list (I. e. `ul` tag) because your social link contains a list of social icons so it will be more semantic to wrap the in a list 👍🏼  e.g.

```
<ul>
     <li>
         <a class="social_media_icon">
            <img src="assets/github.svg" alt="github icon "/>
         </a>
  </li>
       <li>
         <a class="social_media_icon">
            <img src="assets/github.svg" alt="github icon "/>
         </a>
  </li>
</ul>
```